raiden/transfer/log.py

```diff
@@ -192,6 +224,13 @@ def get_all_state_changes(self):
             for res in self.storage.get_all_state_changes()
         ]
 
+    def get_all_state_events(self):
```

Comment:
This interface is using the underlying `get_all_state_events`, this can potentially load too many objects in memory.